hknweb/templates/base.html

```diff
@@ -4,14 +4,18 @@
     <head>
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-  
+
         <!-- Fonts and CSS -->
         <link href="https://fonts.googleapis.com/css?family=Open+Sans|Source+Sans+Pro:300,400,600,700" rel="stylesheet">
         <link rel="stylesheet" href="{% static "css/base.css" %}">
+        <!-- Not sure what this is for? We already have a favicon -->
```

Comment:
Try removing the line and seeing if the favicon sticks around.

Reply:
The favicon still sticks around even if I remove both icons... I think it's cached.